kreuzberg/_document_classification.py

```diff
@@ -50,7 +56,13 @@ def classify_document(result: ExtractionResult, config: ExtractionConfig) -> tup
         A tuple containing the detected document type and the confidence score,
         or (None, None) if no type is detected with sufficient confidence.
     """
-    content_lower = result.content.lower()
+    if not GoogleTranslator:
```

Comment:
I would do the import inside the function simply, or abstract it into its own function-

`get_google_translator` etc.